engine/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -232,13 +234,18 @@ private static Event buildEvent() {
         action.setTimestamp(LocalDateTime.now());
         action.setAccountId("account-id");
         action.setRecipients(List.of());
-        action.setContext(new HashMap<>());
+        action.setContext(new Context.ContextBuilder().build());
         action.setEvents(
                 List.of(
-                        com.redhat.cloud.notifications.ingress.Event
-                                .newBuilder()
-                                .setMetadataBuilder(Metadata.newBuilder())
-                                .setPayload(Map.of("k1", "v1", "k2", "v2", "k3", "v3"))
+                        new com.redhat.cloud.notifications.ingress.Event.EventBuilder()
```

Comment:
Suggestion for a future `insights-notification-schemas-java` release:
```suggestion
                        com.redhat.cloud.notifications.ingress.Event.builder()
```
(with a static `builder` method).

Reply:
This class is autogenerated - Maybe there is an option to have static builders that I didn't see? If not, we could try to ask for support of that feture.

Reply:
I forgot some classes are auto-generated :smiley: